### PR TITLE
add more information in README.md by SinaLale

### PR DESCRIPTION
**Pull request by sinalalebakhsh**
**Checklist**

- [ ] Closing issues: #issue
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [x] I wrote some documentation

**Description**

I hope this is useful to you my friend.
I pull request again. if I can be useful

### CODE REVIEW

File: r2ai/bin/python [hunk 1]
```diff
@@ -0,0 +1 @@
+python3.12
```
Comment (trufae) [inline r2ai/bin/python:1]: What are those files. Looks unrelated

File: README.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ python -m venv r2ai
 . r2ai/bin/activate
 ```
 
+if you have this error, install this:
+The virtual environment was not created successfully because ensurepip is not
+available.  On Debian/Ubuntu systems, you need to install the python3-venv
+package using the following command.
+```
```
Comment (trufae) [inline README.md:43]: I think that “python -v venv r2ai” should work without installing this package

Comment (sinalalebakhsh) [reply README.md:43]: In Ubuntu 22.04.3 LTS .
I got some errors. If you want to add this item to the readme code, please.

**Comment by sinalalebakhsh** [comment]
Thank you for response

File: README.md [hunk 3]
```diff
@@ -40,9 +40,19 @@ if you have this error, install this:
...
+1-
 ```
 apt install python3.10-venv
-
+```
+2-
+```
+pip install -r requirements.txt
+```
+3- If you got error when tokentrim installation:  pip install tokentrim
```
Comment (trufae) [inline README.md:52]: Tokentrim is no longer needed. I removed the dependency because it was buggy and very slow. It was sometimes falling into an infinite loop

**Comment by trufae** [comment]
i have updated the readme, hope the instructions are clearer now :)

thanks for the feedback and testing!
